content/applications/inventory_and_mrp/inventory/product_management/configure/type.rst

```diff
@@ -4,228 +4,349 @@ Product type
+purchase orders. Untracked goods *cannot* be managed using reordering rules.
 
-.. _inventory/product_management/transfer-serv:
+Reordering rules can be configured directly on the product form via the
+:icon:`fa-refresh` :guilabel:`(refresh)` icon.
 
-Service products cannot be included in transfers, but these products can be `linked to projects and
-tasks for deadline tracking <https://www.youtube.com/watch?v=fix2LGkv13c>`_.
+.. note::
+   If reordering rules already exist on a product, Odoo will relabel this button to
+   :guilabel:`Min / Max`, to show the minimum and maximum number of units that must be in stock.
 
 .. _inventory/product_management/po:
 
-Create purchase order
----------------------
+Create purchase orders
+^^^^^^^^^^^^^^^^^^^^^^
 
-Both storable and consumable products can be included in a request for quotation in the *Purchase*
-app.
+Both tracked and untracked products can be included in a request for quotation in the **Purchase**
+app. However, when receiving untracked products, their on-hand quantity does not change upon
+validating the receipt (`WH/IN`).
 
-However, when receiving consumable products, their on-hand quantity does not change upon validating
-the receipt (e.g. `WH/IN`).
+Replenish smart button
+^^^^^^^^^^^^^^^^^^^^^^
 
-.. _inventory/product_management/manufacture:
+The :guilabel:`Replenish` smart button allows all goods to be restocked directly from the product
+form, according to the *Preferred Route*.
 
-Manufacture or subcontract
---------------------------
+.. seealso::
+  :doc:`Replenishment <../../warehouses_storage/replenishment>`
+  `Odoo Tutorials: Replenishment Methods for Manufacturing
+  <https://www.youtube.com/watch?v=vtjeMGcG8aM>`_
 
-Storable and consumable products can be manufactured, subcontracted, or included in a bill of
-materials (BoM).
+.. _inventory/product_management/manufacturing:
 
-.. figure:: type/manufacture.png
-   :align: center
-   :alt: Show "Bill of Materials" and "Used In" smart buttons.
+Manufacturing
+~~~~~~~~~~~~~
 
-   When the **Bill of Materials** and **Used In** smart buttons are visible on the product form,
-   this indicates the product can be manufactured or used as a component of a |BOM|.
+Both tracked and untracked products can be manufactured,
+:doc:`subcontracted <../../../manufacturing/subcontracting>`, or included in another product's
+:doc:`bill of materials (BoM) <../../../manufacturing/basic_setup/bill_configuration>`.
 
-.. _inventory/product_management/package:
+.. _inventory/product_management/BoM:
 
-Packages
---------
+On the product form for a tracked or untracked good, there are several smart buttons that may
+appear for manufacturing operations:
 
-Both storable and consumable products can be placed in :doc:`packages <package>`.
+- :icon:`fa-flask` :guilabel:`Bill of Materials`: This show the BoMs used to make this product.
+- :icon:`fa-level-up` :guilabel:`Used In`: This shows other goods that include this product in their
+  BoM.
```

Comment:
```suggestion
- :icon:`fa-flask` :guilabel:`Bill of Materials`: shows the BoMs used to make this product
- :icon:`fa-level-up` :guilabel:`Used In`: shows other goods that include this product in their
  BoM
```

Reply:
I think I should leave these in for consistency; you had suggested previously that, for the other bullet points introducing icons / reports, I use transition phrases like, "This respresents..." etc.